### ISSUE TITLE
MESH CHECKING HAS FAILED!!!!

### ISSUE THREAD

**Issue by pvh1983** [dfnWorks/dfnWorks-Version2.0#2]
Hello, 

I have played with the dfnWork 2.0 for several weeks to now. The code works great so far and thank you for all your hard work! Right now, I have three questions: 
1. The error of "MESH CHECKING HAS FAILED!!!! Fracture number  399 running on  Process-15". It seems that this an error of parallel implementation for mesh generation. However, I did test the code for 1 CPU but still seeing this error. What should I do go overcome this? 
2. A wrong explanation for rAngleOption for the input file for dfnGen. rAngleOption should be 1 for degree and 0 for radians (I guess). 
3. Is there a way to run the dfnWorks 2.0 using run.py without using "$DISPLAY environment variable" option? If I login with ssh -X  [SERVER_NAME] and run the code in the head node, it works perfect. But when I submit the job, it didn't work. Therefore, I have to use run_explicit.py but the output is not as well-organized as running model with run.py and I don't even know if there is any difference between run.py and run_explicit.py.

Thank you, 
Hai.

**Comment by deknapp** [comment]
@pvh1983 

First, have you cloned the latest version (pushed today at about 1 pm? Some meshing issues have been resolved with this commit. Try doing:

$ cd ~/dfnWorks-Version2.0/pydfnworks/bin/
$ python test.py rect

If that doesn't work, please respond on this thread re your point 1. 

2. This has been noted and will be corrected in our next release. 

3. run_explicit.py allows you to run only certain parts of the suite. Try commenting out line 24:
DFN.output_report(), and this should solve the $DISPLAY issue. 

I will commit a change shortly that cleans up the output in run_explicit.py

Thanks,
Nathan

**Comment by deknapp** [comment]
The output should be cleaner now when using run_explicit.py.

**Comment by pvh1983** [comment]
Thanks Nathan for fixing the error of "MESH CHECKING HAS FAILED" and clean the outputs. The model runs a lot better than before. After a lot of simulations, I found that the code works well for a system of around 2,000 fractures given a model domain of 100x200x100m and P32 around 0.4 (m2/m3). However, when increasing P32 or the number of fractures, I  found three more issues:

(1) I found this error when asking a network of 20,000 fractures: "ERROR! Incorrect Number of points removed". More details are in the attached file: [hpham81.txt](https://github.com/dfnWorks/dfnWorks-Version2.0/files/926914/hpham81.txt). I still see the full_mesh.inp but no flow and transport results.

(2) There is a situation that the dfnGen could not produce fracture networks when it is asked for stopCondition: 1 (Stop based on family's p32 values). The model kept running but nothing showed up except the file radii_All.dat (which is kept growing up). I guess the algorithm couldn't produce a network that satisfies 20 P32 values (that come from 4 model layers and 5 fracture families). What should I do in this case? 

(3) Sometimes, Paraview could not read *.inp in traj/trajectories. I have tried several versions of Paraview and still seeing the error. It is noted that Paraview could read the *.inp in traj/trajectories outputs from the test cases. I attached an example of and inp file that I couldn't open here as well.
[part_784.inp.txt](https://github.com/dfnWorks/dfnWorks-Version2.0/files/926934/part_784.inp.txt)


Thank you, 
Hai.
